routes/users.js

```diff
@@ -0,0 +1,95 @@
+const express = require('express');
+const router = express.Router();
+const knex = require('../database');
+
+router
+  .route('/:user_id')
+  .get((req, res) => {
+    knex
+      .raw('select * from users where id = ?', [req.params.user_id])
+      .then(function(userObject) {
+        if (userObject.rows.length === 0) {
+          return res.status(404).send('{ "message": "User not found!" }');
+        }
+
+        return res.status(200).send(userObject.rows);
```

Comment:
You should return the user object itself instead of an array of just one object.